CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## 1.27.1
```

Comment:
We should start of by marking this as unreleased. Selecting a version is a release process decision that we need to make (which might even be a feature?).